src/Pyz/Yves/HelloSpryker/Plugin/Provider/HelloSprykerControllerProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * This file is part of the Spryker Commerce OS.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Pyz\Yves\HelloSpryker\Plugin\Provider;
+
+use Silex\Application;
+use SprykerShop\Yves\ShopApplication\Plugin\Provider\AbstractYvesControllerProvider;
+
+class HelloSprykerControllerProvider extends AbstractYvesControllerProvider
```

Comment:
You defined 2 classes for registering your new route, but you only need one, please remove the not used class.